Framework/WorkflowAlgorithms/src/AlignAndFocusPowder.cpp

```diff
@@ -57,6 +57,7 @@ const std::string RESAMPLEX("ResampleX");
 const std::string BIN_IN_D("Dspacing");
 const std::string D_MINS("DMin");
 const std::string D_MAXS("DMax");
+const std::string DELTA("Delta");
```

Comment:
Please rename this to `RaggedDelta` or `DeltaRagged` so people more easily understand what it does without reading the docs.